tests/phpunit/includes/links-trait.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * Trait to ease tests made with WP installed in a subfolder.
+ */
+trait PLL_Links_Trait {
```

Comment:
To avoid a potential naming conflict in the future:
```suggestion
trait PLL_Test_Links_Trait {
```

Reply:
Done in 81d94c08.